### PR TITLE
Update vulcan-lucky to v2

### PR DESCRIPTION
**Pull request by octomit**
Update vulcan-lucky to v2

### CODE REVIEW

File: cmd/vulcan-lucky/main.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 Adevinta
+Copyright 2022 Adevinta
```
Comment (julianvilas) [inline cmd/vulcan-lucky/main.go:2]: I think this is not required. We would put the 2022 header just if the file has been created on 2022. But I might be wrong.

File: cmd/vulcan-lucky/main.go [hunk 2]
```diff
@@ -29,10 +29,13 @@ var (
...
 		Description: "AES-NI implementation in OpenSSL before 1.0.1t and 1.0.2 before 1.0.2h allows remote attackers to obtain sensitive" +
 			"cleartext information",
-		Score:           report.SeverityThresholdMedium,
-		ImpactDetails:   "Allows remote attackers to obtain sensitive cleartext information via a padding-oracle attack against an AES CBC session",
-		References:      []string{"https://blog.cloudflare.com/yet-another-padding-oracle-in-openssl-cbc-ciphersuites/"},
-		Recommendations: []string{"Upgrade OpenSSL to, at least, 1.0.2h or 1.0.1t"},
+		AffectedResource: "443/tcp",
+		Score:            report.SeverityThresholdMedium,
+		Fingerprint:      helpers.ComputeFingerprint(),
+		Labels:           []string{"issue"},
```
Comment (julianvilas) [inline cmd/vulcan-lucky/main.go:35]: I'd suggest also adding the optional `ssl` label, as we have added it in some other checks reporting problems related to SSL.

**Review by julianvilas — COMMENTED** [review]
Very good job man! thx for the contribution. Just a couple of minor suggestions

File: cmd/vulcan-lucky/main.go (hunk 1 above)
Comment (octomit) [reply cmd/vulcan-lucky/main.go]: Addressed in 83d2167c389d0f4be6f076ccd271a29ad32e1c77

File: cmd/vulcan-lucky/main.go (hunk 2 above)
Comment (octomit) [reply cmd/vulcan-lucky/main.go]: Done. 
83d2167c389d0f4be6f076ccd271a29ad32e1c77

**Review by julianvilas — APPROVED** [review]

**Pull request merged**